### DISCUSSION TITLE
403 on Patch

### DISCUSSION THREAD

**Discussion by ewingson** [codecentric/web-clip#33] [Q&A]
![Bildschirmfoto_2023-01-09_06-49-12](https://user-images.githubusercontent.com/36561948/211247694-b552030c-5e46-4f5e-8793-ac4f4e5cbba2.png)

hi Angelo.

I get the following error

back in 2021 everything worked 
![2](https://user-images.githubusercontent.com/36561948/211248136-de5bfa5c-f7bd-4a7c-a106-15514cc86cea.png)
well. an idea ?

**Comment by ewingson** [comment]
now I got

```
Outline.expand: Unable to fetch <https://ewingson.solidcommunity.net/webclip/2023/01/09/2eb78bb1-8897-45c3-b0ce-d67981290974>: Failed to load  <https://ewingson.solidcommunity.net/webclip/2023/01/09/2eb78bb1-8897-45c3-b0ce-d67981290974> Fetcher: Error trying to parse <https://ewingson.solidcommunity.net/webclip/2023/01/09/2eb78bb1-8897-45c3-b0ce-d67981290974> as Notation3:
SyntaxError: Line 26 of <https://ewingson.solidcommunity.net/webclip/2023/01/09/2eb78bb1-8897-45c3-b0ce-d67981290974>: Bad syntax: expected '.' or '}' or ']' at end of statement
at: "E1;
    schema:priceCurrency "" status: 200
```

**Comment by ewingson** [comment]
@angelo-v 

solved.

it happened to be a formatting problem with the displayed price. they were using `E01`-syntax. when correcting to `24.99` everything went fine.

see https://ewingson.solidcommunity.net/webclip/2023/01/09/a29da338-9e66-4e5d-b6df-c71416f37049

**Reply by angelo-v** [reply]
I fixed a similar bug in rdflib already, perhaps you can file an issue there. E-Notations should actually be supported. For WebClip we need to improve the error handling

**Reply by angelo-v** [reply]
I am not allowed to see your clip, you need to give access to my WebID or public access

**Reply by angelo-v** [reply]
Can you provide the URL of the page you tried to clip?

**Comment by ewingson** [comment]
now I experience a seemingly *hidden* file.

see screenshot, although https://ewingson.solidcommunity.net/webclip/2023/01/09/a29da338-9e66-4e5d-b6df-c71416f3704 exists.

I consider this a strange side-effect
![Bildschirmfoto_2023-01-09_18-21-57](https://user-images.githubusercontent.com/36561948/211369380-f5d537be-6485-4f01-a06f-f0f3f05df928.png)

**Reply by angelo-v** [reply]
This is a feature of SolidOS, where it tries to show the `#this` resource by default when opening a container with an `index.ttl` but WebClip does not have such a resource in the index. You need to directly open index.ttl to see its contents

**Comment by ewingson** [comment]
@angelo-v 
(I added my comment beneath the thread)

a) I gave you editor-access (read and change)

![Bildschirmfoto_2023-01-09_21-07-22](https://user-images.githubusercontent.com/36561948/211398497-a20cd748-a6d7-4b1f-babe-27a4de933412.png)

b) the URL of the clipped page is https://www.decathlon.de/p/badmintonschlager-br-530-erwachsene/_/R-p-144872?mc=8626422

c) changing the E-Notation to decimal fixed it, so I don't think we got a problem there

d) the clipped thang is in https://ewingson.solidcommunity.net/webclip/2023/01/09/a29da338-9e66-4e5d-b6df-c71416f37049

**Reply by angelo-v** [reply]
My WebID is https://angelo.veltens.org/profile/card#me ;)

**Comment by ewingson** [comment]
now I dragged both https://aveltens.solidcommunity.net and https://aveltens.solidcommunity.net/profile/card#me to the Editors but it only displays one of them ;-)

**Reply by angelo-v** [reply]
You should remove both and add my real WebID https://angelo.veltens.org/profile/card#me

**Comment by ewingson** [comment]
did that. added https://angelo.veltens.org/profile/card#me but just displays the name. hope it's functional. I must have been unattentive for a moment.
